src/components/subscriptions/data/hooks.js

```diff
@@ -54,16 +54,16 @@ export const useSubscriptions = ({ enterpriseId, errors, setErrors }) => {
         setSubscriptions(subscriptionsData);
       } catch (err) {
         logError(err);
-        setErrors({
-          ...errors,
+        setErrors(s => ({
+          ...s,
```

Comment:
Use the callback function to retrieve the current state rather than relying on the current `errors` state variable (anti-pattern).